data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_inventor.json

```diff
@@ -127,7 +127,7 @@
     "material": [ "forged_dreamstuff" ],
     "symbol": "?",
     "looks_like": "textbook_anarch",
-    "scannable": false,
+    "//scannable": [ false, "not a book, but shouldn't be scannable" ],
```

Comment:
Is this how making something not scannable should work or does the not scannable object not exist?

Reply:
These are not books, so they can't be scanned. I'm just keeping this as a comment should that change in the future.